### ISSUE TITLE
learning rate output mismatched with the desired one

### ISSUE THREAD

**Issue by s4ki3f** [ultralytics/ultralytics#445]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

I was training a model for classification for fine tuning I was changing some hyperparameters and found this.
`!yolo task=classify mode=train model=/content/runs/classify/train2/weights/last.pt data='/content/Cow-Identification-1' epochs=3 lr0=0.01 lrf=0.001`
this was my input and the output should have shown this change but it did not

![image](https://user-images.githubusercontent.com/29111757/212922673-f7fc64f5-44b9-4552-8997-ee6de7c04650.png)

but, I figured it out, it is as it is inputted on the terminal in args.yaml file
![image](https://user-images.githubusercontent.com/29111757/212923066-94a08f0c-d5f8-45e4-81bf-5f0508273874.png)

When I am changing the optimizer it shows right information but the learning rate details does not provide right details

![image](https://user-images.githubusercontent.com/29111757/212924582-ed1f7067-f3d5-461a-82be-a790f3a2d490.png)
`!yolo task=classify mode=train model=/content/runs/classify/train2/weights/last.pt data='/content/Cow-Identification-1' epochs=3 lr0=0.01 lrf=0.001 optimizer=Adam`
![image](https://user-images.githubusercontent.com/29111757/212924702-168f1f4d-fd75-4fac-a153-e325d25d0ab6.png)

### Environment

Ultralytics YOLOv8.0.6 🚀 Python-3.8.16 torch-1.13.0+cu116 CUDA:0 (Tesla T4, 15110MiB)
On google colab

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by AyushExel** [comment]
Right now classification task has fixed lr and some other params. We'll fix that soon.

**Comment by glenn-jocher** [comment]
@s4ki3f thanks for the bug report! Your issue should be resolved now in #476 which will be part of 8.0.10 deployment later today. Let us know if this resolves your issue and if you find any other problems!
